deploy/k8s/README.md

```diff
@@ -0,0 +1,200 @@
+# K8s Deploy
+This [deploy.yaml](deploy.yaml) demonstrate how to run subquery in k8s.
+
+You will need to run 
+### 1. a subql-node to index data
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
in my opinion, we can use deployment for the indexer, coz we use in-memory volume here, it will be destroyed after the pod was gone.

Reply:
good point